html5/default/util/index.js

```diff
@@ -49,6 +49,20 @@ else {
 export { _Set }
 
 /**
+ * Polyfill in iOS7 by native because the JavaScript polyfill has memory problem.
+ * @return {object}
+ */
+
+export function createNewSet () {
+  /* eslint-disable */
+  if (typeof nativeSet === 'object') {
+    return nativeSet.create()
+  }
+  /* eslint-enable */
+  return new Set()
```

Comment:
Notice that the `Set` is from ES6, you should use `_Set` here ( It's a polyfill of `Set` on the above).